hangupsbot/plugins/default.py

```diff
@@ -48,7 +48,12 @@ def echo(bot, event, *args):
 
         yield from command.run(bot, event, *["convecho", "id:" + convid, _text])
 
-
+def admins(bot,event,*args):
+    """Get a list of bot admins"""
+    admins = ' <br>'.join([bot.get_hangups_user(uid).full_name for uid in bot.get_config_option('admins')])
```

Comment:
That works.  Thanks, should be all set to go.

One of these days i'll sit down and actually learn python as opposed to what i'm doing now :-)